sgp4/propagation.py

```diff
@@ -1680,11 +1680,9 @@ def sgp4init(
   ----------------------------------------------------------------------------*/
 """
 
-def sgp4(satrec, tsince, whichconst=None):
+def sgp4(satrec, tsince):
```

Comment:
This also looks like it would break old code. Can we revert and restore compatibility?

Reply:
See previous comment.

Reply:
Reverted to original code with recent changes to whichconst.